dummyserver/handlers.py

```diff
@@ -217,7 +217,6 @@ def echo_params(self, request):
     def sleep(self, request):
         "Sleep for a specified amount of ``seconds``"
         # DO NOT USE THIS, IT'S DEPRECATED.
-        # FIXME: Delete this once appengine tests are fixed to not use this handler.
```

Comment:
Can you please remove the whole function? As noted in the comment you rightly removed, it was only used by the App Engine tests. (You'll need to remove the `time` import too.)

Reply:
Unresolving as it's not done yet, sorry.